### ISSUE TITLE
Support for code and Math equations

### ISSUE THREAD

**Issue by sgondala** [nuttyartist/notes#115]
It's useful if we can add support for code snippets (As we see it in gitter, for eg) and math equations.

**Comment by nuttyartist** [comment]
Yes, that will be good (support for code), don't how about Math. We'll get into that when we'll implement support for rich text.
